### PR TITLE
Update README.md to include protocgen example

### PR DESCRIPTION
**Pull request by MuweiHe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,15 @@
 
 > Public Golang SDK for Confluent Cloud.
 
+## To generate / update `*pb.go` files
+
+* Install `protoc-gen-go`,
```
Comment (brianstrauch) [inline README.md:7]: Can you write out the full command?

File: README.md [hunk 2]
```diff
@@ -2,3 +2,15 @@
 
 > Public Golang SDK for Confluent Cloud.
 
+## To generate / update `*pb.go` files
+
+* Install `protoc-gen-go`, 
+
+* run `go mod vendor` in `ccloud-sdk-go-v1-public`
```
Comment (brianstrauch) [inline README.md:9]: I think it's already assumed we're in this repo.
```suggestion
* Run `go mod vendor`
```

**Review by sgagniere — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -2,3 +2,15 @@
...
+
+* at `ccloud-sdk-go-v1-public` root dir, run the following command:
```
Comment (brianstrauch) [inline README.md:11]: Make a note to swap out `org/org.proto` with the relevant file?

File: README.md [hunk 4]
```diff
@@ -4,13 +4,18 @@
...
+* Install `protoc-gen-go`:
+```
+go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
+```
 
-* run `go mod vendor` in `ccloud-sdk-go-v1-public`
+* Run `go mod vendor`
 
-* at `ccloud-sdk-go-v1-public` root dir, run the following command:
+* At `ccloud-sdk-go-v1-public` root dir, run the following command, replace `org/org.go` with relevant file
```
Comment (brianstrauch) [inline README.md:14]: I think the command above also implies that we're in the root directory, so it should be fine to leave this out.
```suggestion
* Run the following command, replacing `org/org.go` with the relevant file
```

File: README.md [hunk 5]
```diff
@@ -4,13 +4,18 @@
...
 ```
 protoc --go_out=. --go_out=/ org/org.proto -I=./ -I=vendor/github.com/gogo/protobuf \
   -I=vendor/github.com/envoyproxy/protoc-gen-validate \
   -I=vendor 
 ```
+
+* Remove vendor folder and make a PR
```
Comment (brianstrauch) [inline README.md:21]:
```suggestion
* Remove the `vendor/` directory and open a PR
```

**Review by brianstrauch — APPROVED** [review]

**Pull request merged**